docs/authentication.md

```diff
@@ -255,7 +252,10 @@ Privileged session expire after a set amount of time. To log out manually, issue
 
 To alleviate the need for supplying authentication in the command-line every time a server is connected to, users can pre-configure their credentials to be used in authentication.
 
-To do so, open `~/.codechecker_passwords.json`. The `credentials` section is used by the client to read pre-saved authentication data in `username:password` format.
+To do so first copy the `config/session_client.json` file from the CodeChecker package to your home directory and rename it to `codechecker_passwords.json`
+After creating the new file open `~/.codechecker_passwords.json`.
```

Comment:
Shouldn't we warn the users about setting the permissions properly in case we do not set it?